sopel/modules/etymology.py

```diff
@@ -40,8 +40,11 @@ def etymology(word):
     # @@ <nsh> sbp, would it be possible to have a flag for .ety to get 2nd/etc
     # entries? - http://swhack.com/logs/2006-07-19#T15-05-29
 
+    if len(word) == 0:
+        return 'No word added.'
```

Comment:
I'd suggest to raise a Value error here instead:

```suggestion
        raise ValueError('No word to look for.')
```